css/popup.css

```diff
@@ -1,177 +1,57 @@
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
 }
-#profile_box div {
-    float: left;
-    padding: 15px;
-    padding-bottom: 0px;
+
+.app-icon {
+    height: 25px;
+    width: 25px;
+
+    border-radius: 0;
+
+    margin-bottom: 0;
 }
-.profile_pic {
-   height: 65px;
+
+#loader, #login, #main {
+    height: 100%;
 }
+
 #main {
-    width: 390px;
-    overflow: auto;
-    padding: 10px 0;
-}
-#user_name_box {
-    color: #3988AD;
-    height: 60px;
-    font-size: 18px;
-    /* padding: 10px; */
-    /* padding-bottom: 0; */
-    padding-left: 15px; 
-}
-#user_box {
-    height: 84px;
-}
-p#user_name {
-    width: 190px;
-    margin-left: -15px;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+    padding: 8px;
 }
-p#user_info {
-    font-size: 13px;
-    padding-top: 7px;
-    margin-left: -14px;
-    color: #797979;
-    width: 190px;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-}
-.img img {
-    width:35px;
-    height: 35px;
-    margin-left: 4px;
-}
-.text {
-    color: #999;
-    font-size: 16px;
-    padding: 6px 10px;
-}
-.links {
-    overflow:auto;
-    width: 185px;
-    cursor:pointer;
-    border-bottom: 1px solid #eee;
-    float:left;
-    padding-top: 6px;
-    padding-bottom: 2px;
-    margin-left: 10px;
-}
-.links:hover {
-    background-color: #F0F0F0;
-}
-.link1 {
-    border-top: 1px solid #eee;
-}
-.links div {
-    float:left;
-}
-#submit {
-    background: url(../images/submit.png) no-repeat;
-    border: none;
-    width: 38px;
-    height: 62px;
-    float: right;
-    margin-top: -67px;
-    z-index: 100;
-    position:relative;
-    cursor:pointer;
-    outline:none;
-    opacity: 0.9;
-}
-#submit:hover {
-    opacity:1;
-}
-#loggedin{
-    text-align: center;
-    color: #555;
-    /* padding: 15px 10px; */
-    height: 45px;
-    /* float: right; */
-    /* margin-top: -43px; */
-    position: absolute;
-    right: 2px;
-    top: 4px;    
-}
-::-webkit-input-placeholder {
- 	  color: #ccc;
+
+#loading-gif {
+    height: 16px;
+    width: 16px;
 }
-#logout_btn{
-    border: none;
-    color: #b1b1b1;
-    background-color: rgb(255, 255, 255);
-    padding: 5px 20px;
-    border-radius: 3px;
-    font-size: 16px;
-    margin: 10px;
-    /* -webkit-box-shadow: 0px 1px 1px 0px rgba(50, 50, 50, 0.75); */
-    -moz-box-shadow: 0px 1px 1px 0px rgba(50, 50, 50, 0.75);
-    /* box-shadow: 0px 1px 1px 0px rgba(50, 50, 50, 0.75); */
-    cursor: pointer;
-    border: 1px solid #D6D6D6;
+
+#img-logo-img {
+    height: 125px;
+    width: 125px;
 }
-#logout_btn:hover{
-    background: rgb(248, 248, 248);
+
+#user-photo {
+    height: 48px;
+    width: 48px;
+
+    margin-bottom: 0;
 }
-#footer {
-    display: block;
-    width: 100%;
-    height: 40px;
-    padding-top: 5px;
+
+#channel-i-logo {
+    height: 30px;
+    width: 80px;
 }
+
+#img-logo-footer {
+    height: 30px;
+    width: 30px;
+}
```

Comment:
Try to make this css cross-browser compatible (atleast for Google Chrome, Mozilla Firefox and Safari)